spec/toggles/init_spec.rb

```diff
@@ -49,7 +49,10 @@
       Toggles.init
 
       expect(Feature::Foo::Users.enabled_for?(id: 1)).to eq(false)
-      expect { Feature::Foo::Children.enabled_for?(id: 1) }.to raise_error(NameError)
+      expect { Feature::Bar::Children.enabled_for?(id: 1) }
+        .to raise_error(Feature::ConstantLookup::Error, 'Feature::Bar')
+      expect { Feature::Foo::Children.enabled_for?(id: 1) }
+        .to raise_error(Feature::ConstantLookup::Error, 'Feature::Foo::Children')
```

Comment:
actually thinking about this change, `NameError` -> `Feature::ConstantLookup::Error`, this could screw users up. I guess it's a pre v1 release so there's no guarantees yet on the API. Any thoughts on this?

Reply:
you're right, `NameError` is the more appropriate parent class.